platform/commonUI/general/src/directives/MCTTree.js

```diff
@@ -22,9 +22,10 @@
 
 define([
     'angular',
-    '../ui/TreeView'
-], function (angular, TreeView) {
-    function MCTTree(gestureService) {
+    '../ui/TreeView',
+    '../ui/TreeNodeView'
+], function (angular, TreeView, TreeNodeView) {
```

Comment:
No need to add a dependency on the `TreeNodeView` here. Pass `openmct` into the constructor for `TreeView`, then modify `TreeView` to pass `openmct` to the constructor for `TreeNodeView`.

Reply:
Hi @akhenry, `openmct` comes to be undefined into the TreeView.
I also observed that the TreeView is not present in bundle.js, so that I can insert openmct as a dependency.
Will you please check this?